include/nighthawk/adaptive_load/metrics_plugin.h

```diff
@@ -1,3 +1,5 @@
+#pragma once
```

Comment:
As above, the second #pragma once needs to be removed.

Reply:
Done in 78338dca6de640193a4ea395ed2052c832397ea8.